python/pyarrow/_parquet.pyx

```diff
@@ -35,6 +35,271 @@ from pyarrow.lib import ArrowException, NativeFile
+        lines = [prefix+line for line in text.splitlines(True)]
+        return ''.join(lines)
+
+
+cdef class EncodedStatistics:
+    cdef:
+        CEncodedStatistics statistics
+
+    def __cinit__(self):
+        pass
+
+    cdef init(self, const CEncodedStatistics& statistics):
+        self.statistics = statistics
+
+    def __repr__(self):
+        return """{0}
+  max: {1}
+  min: {2}
+  null_count: {3}
+  distinct_count: {4}
+  has_min: {5}
+  has_max: {6}
+  has_null_count: {7}
+  has_distinct_count: {8}""".format(object.__repr__(self),
+                                    self.max,
+                                    self.min,
+                                    self.null_count,
+                                    self.distinct_count,
+                                    self.has_min,
+                                    self.has_max,
+                                    self.has_null_count,
+                                    self.has_distinct_count)
+
+    property max:
+
+        def __get__(self):
+            return self.statistics.max()
+
+    property min:
+
+        def __get__(self):
+            return self.statistics.min()
+
+    property null_count:
+
+        def __get__(self):
+            return self.statistics.null_count
+
+    property distinct_count:
+
+        def __get__(self):
+            return self.statistics.distinct_count
+
+    property has_min:
+
+        def __get__(self):
+            return bool(self.statistics.has_min)
+
+    property has_max:
+
+        def __get__(self):
+            return bool(self.statistics.has_max)
+
+    property has_null_count:
+
+        def __get__(self):
+            return bool(self.statistics.has_null_count)
+
+    property has_distinct_count:
+
+        def __get__(self):
+            return bool(self.statistics.has_distinct_count)
+
+
+cdef class RowGroupStatistics:
+    cdef:
+        shared_ptr[CRowGroupStatistics] statistics
+
+    def __cinit__(self):
+        pass
+
+    cdef init(self, const shared_ptr[CRowGroupStatistics]& statistics):
+        self.statistics = statistics
+
+    def __repr__(self):
+        return """{0}
+  null_count: {1}
+  distinct_count: {2}
+  num_values: {3}
+  physical_type: {4}""".format(object.__repr__(self),
+                               self.null_count,
+                               self.distinct_count,
+                               self.num_values,
+                               self.physical_type)
+
+    def Reset(self):
+        self.statistics.get().Reset()
+
+    def SetComparator(self):
+        self.statistics.get().SetComparator()
+
+    def HasMinMax(self):
+        return bool(self.statistics.get().HasMinMax())
+
+    def EncodeMin(self):
+        return self.statistics.get().EncodeMin()
+
+    def EncodeMax(self):
+        return self.statistics.get().EncodeMax()
+
+    def Encode(self):
+        statistics = EncodedStatistics()
+        statistics.init(self.statistics.get().Encode())
+        return statistics
```

Comment:
Can you use snake case for these functions? `encode_min` etc. If these functions are not useful there's no immediate need to expose them